strapi/design-system
# feat : add new code block svg file in icons folder

### What does it do?

This PR adds a new "Code block" icon to the icon library used in Strapi. The new icon will allow for clearer differentiation between "Code block" and inline "Code" text modifiers.

### Why is it needed?

The addition of this icon addresses user feedback indicating confusion between code blocks and inline code. Having a dedicated "Code block" icon will improve the user experience by making it easier to identify the block element, which is distinct from an inline modifier, and aligns with design consistency.

### How to test it?

1. **Storybook Verification** : Check the Storybook instance for the icon library. The `CodeBlock` icon should be visible along with other icons, as shown in the screenshot.

![Storybook Icon Screenshot](https://github.com/user-attachments/assets/1e9e5527-1a4b-4948-9481-f4f9441ae4c1)

2. **Visual Inspection** : Ensure the `CodeBlock` icon matches the intended design and is distinguishable from the `Code` icon.

### Related issue(s)/PR(s)

This PR closes issue #1804 by introducing the "Code block" icon from the Phosphor library and adding it to the design system as part of the icon update process in Strapi 5.
